### ISSUE TITLE
rPi4 needs --led-slowdown-gpio=2, should it be a default?

### ISSUE THREAD

**Issue by marcmerlin** [hzeller/rpi-rgb-led-matrix#996]
~/rpi-rgb-led-matrix3/examples-api-use/demo --led-gpio-mapping=regular --led-rows=64 --led-cols=128  --led-row-addr-type=0 --led-chain=1 --led-show-refresh --led-slowdown-gpio=1 --led-parallel=3   --led-pwm-dither-bits=1 --led-pwm-lsb-nanoseconds=50 --led-pwm-bits=7 --led-panel-type=FM6126A -D0

did not give me a workable display (670.8Hz), this is using the code built on rPi3, which was a bit faster than the same code rebuilt on rPi4  (true for --led-slowdown-gpio=1 but with --led-slowdown-gpio=2 then the rPI4 rebuilt code is actually faster)

![IMG_20200305_091221](https://user-images.githubusercontent.com/1369412/76007227-908c4680-5ec2-11ea-8435-dc212c6fc337.jpg)

**Comment by marcmerlin** [comment]
![IMG_20200305_091103](https://user-images.githubusercontent.com/1369412/76007259-a0a42600-5ec2-11ea-8c4c-602c0bb5e07a.jpg)

**Comment by marcmerlin** [comment]
With --led-slowdown-gpio=2, it works without issues.
As a bonus, ABCDE panels (128x64) in led-parallel=3, work a bit faster than on rPI3 (411.7Hz vs 373.3Hz)

**Comment by hzeller** [comment]
yes, i am considering making the default dependent on the detected pi.

**Comment by marcmerlin** [comment]
With apologies to everyone, this project is doing a one time cleanup of 
all the bugs that have not been addressed and closed in the last 10 years.

If you filed an issue to ask for help, please do not re-open this one or a new
bug, but go here instead: https://rpi-rgb-led-matrix.discourse.group/

Please be mindful that this project does not have any full time maintainers
or anyone paid. You should not expect personal help for your own projects or
answers to programming questions but you can ask for help with panels that
aren't working, setup questions, or anything that may be a bug here:
https://rpi-rgb-led-matrix.discourse.group/
If you need programming help and are willing to pay someone for it, please
state so in your message, and someone may be interested.

If you did report an actual bug that is still not fixed today, please accept
our apologies for this being mass closed, you can absolutely send a followup
and re-open the issue, or open a new up to date one as applicable.
Note that we very much welcome help, so if you can help fix your bug, you will
be a superstar :)

If your bug was linked to a pull request that was not resolved, please update
the pull request and the issue, and we will do our best to review them. We do
thank you for your undertanding that we have very limited free time, so any
extra work you do along with your bug report or PR request will be very much
appreciated and improve the odds that they will be looked at.

If you want to help contribute as a python or C# or any other language bindings,
you are also a superstar, please state so in the issue and we'll do our best to
give you the access and reviews you need.

For anything related to PWM panel support, please make sure you read README and
then refer to https://github.com/hzeller/rpi-rgb-led-matrix/issues/466 . 
Please do not comment if you're haven't read everything and/or if you're just
asking when your panel is going to be supported. Those bugs are if you're 
able to contribute to the effort.

For rPI5: https://github.com/hzeller/rpi-rgb-led-matrix/issues/1603

For anything python, please link your bug to: https://github.com/hzeller/rpi-rgb-led-matrix/issues/1828
For anything C#, please link your bug to: https://github.com/hzeller/rpi-rgb-led-matrix/issues/1828
